recipe/meta.yaml

```diff
@@ -38,3 +40,4 @@ extra:
   recipe-maintainers:
     - pvanheus
     - thebestnom
+    - xylar
```

Comment:
I definitely don't need to be a maintainer if you'd prefer not to have me.  Just volunteering if you could use a hand.